frontend/src/components/forms/Dropdown.tsx

```diff
@@ -64,12 +67,13 @@ export const Dropdown: FC<{
   }
 
   return (
-    <div className={classes(css.dropContainer)}>
+    <div className={classes(css.dropdownContainer)}>
       <button
         type="button"
         className={classes(css.dropButton)}
         onClick={showMenu}
         disabled={disabled}
+        id={id}
```

Comment:
Now that the `id` is added to this, the dropdown can be opened by clicking the corresponding label.

That's totally fine, but now the dropdown can be closed by clicking practically anywhere on the page *except the label* (which is a bit odd user experience). This could be avoided by preventing the default on the closing click handler.